### PR TITLE
Bugfix/intl remove generated file and clean the code! Intl is now working again

### PR DESCRIPTION
**Pull request by apomalyn**
Switch from the intl_util code generation to flutter gen. 

This PR fixes the intl issue we were facing since the migration to flutter 2.0.0.

### CODE REVIEW

File: README.fr.md [hunk 1]
```diff
@@ -44,12 +44,12 @@ flutter pub get
 flutter pub run intl_utils:generate
```
Comment (AlexandreMarkus) [inline README.fr.md:44]: I think this line should be removed
```suggestion

```

Comment (AlexandreMarkus) [reply README.fr.md:44]: same for Readme.md file

File: lib/l10n/intl_fr.arb [hunk 2]
```diff
@@ -31,7 +31,7 @@
   "login_info_universal_code": "Votre code d'accès universel est sous la forme AB12345. Celui-ci vous a été transmis par le Bureau du registraire lors de votre admission.",
   "login_msg_logout_success": "Vous vous êtes déconnecté",
   "login_applets_logo": "Conçue par",
-  "password_forgotten": "Mot de passe oublié?",
+  "login_password_forgotten": "Mot de passe oublié?",
```
Comment (AlexandreMarkus) [inline lib/l10n/intl_fr.arb:34]: make sure to make these changes to intl_en.arb also

**Review by AlexandreMarkus — CHANGES_REQUESTED** [review]
seems good!!! make sure to address the 2 comments for Readme and arb files

File: lib/l10n/intl_fr.arb (hunk 2 above)
Comment (apomalyn) [reply lib/l10n/intl_fr.arb:34]: This label is already on intl_en.arb :) the label name was different on intl_fr.arb

File: README.fr.md (hunk 1 above)
Comment (apomalyn) [reply README.fr.md]: Done!

**Review by AlexandreMarkus — APPROVED** [review]
nice catch with this upgrade!!

**Pull request merged**